app/src/main/java/com/kickstarter/features/pledgedprojectsoverview/ui/PledgedProjectsOverviewActivity.kt

```diff
@@ -1,5 +1,72 @@
 package com.kickstarter.features.pledgedprojectsoverview.ui
 
+import android.os.Build
+import android.os.Bundle
+import androidx.activity.OnBackPressedCallback
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.lazy.rememberLazyListState
+import androidx.compose.ui.Modifier
+import com.kickstarter.features.pledgedprojectsoverview.viewmodel.PledgedProjectsOverviewViewModel
+import com.kickstarter.libs.utils.TransitionUtils
+import com.kickstarter.libs.utils.extensions.getEnvironment
+import com.kickstarter.libs.utils.extensions.isDarkModeEnabled
+import com.kickstarter.ui.SharedPreferenceKey
+import com.kickstarter.ui.activities.AppThemes
+import com.kickstarter.ui.compose.designsystem.KickstarterApp
+import com.kickstarter.ui.extensions.transition
 
-class PledgedProjectsOverviewActivity : AppCompatActivity()
+class PledgedProjectsOverviewActivity : AppCompatActivity() {
+
+    private lateinit var viewModelFactory: PledgedProjectsOverviewViewModel.Factory
+    private val viewModel: PledgedProjectsOverviewViewModel by viewModels { viewModelFactory }
+    private var theme = AppThemes.MATCH_SYSTEM.ordinal
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val env = this.getEnvironment()?.let { env ->
+            viewModelFactory = PledgedProjectsOverviewViewModel.Factory(env)
+
+            theme = env.sharedPreferences()
+                ?.getInt(SharedPreferenceKey.APP_THEME, AppThemes.MATCH_SYSTEM.ordinal)
+                ?: AppThemes.MATCH_SYSTEM.ordinal
+            env
+        }
+
+        setContent {
+
+            val darkModeEnabled = env?.let { this.isDarkModeEnabled(env = it) } ?: false
+            val lazyListState = rememberLazyListState()
+
+            KickstarterApp(
```

Comment:
nitpick: We can simplify a bit here by using extension `Context.isDarkModeEnabled(env: Environment)` living in `ContextExt.kt`. You can see it being used in OAuthWebViewActivity:49